authenticate.php

```diff
@@ -1,4 +1,5 @@
 <?php
+@session_set_cookie_params(365*24*3600);
```

Comment:
Assuming that `.user.ini` gets run when the web server starts (you could use `phpinfo()` to verify the settings are correct) I don't think this is necessary.